examples/petsc/area.c

```diff
@@ -185,15 +185,16 @@ int main(int argc, char **argv) {
     ierr = PetscPrintf(comm,
                        "\n-- libCEED + PETSc Surface Area of a Manifold --\n"
                        "  libCEED:\n"
-                       "    libCEED Backend                    : %s\n"
-                       "    libCEED Backend MemType            : %s\n"
+                       "    libCEED Backend                         : %s\n"
+                       "    libCEED Backend MemType                 : %s\n"
                        "  Mesh:\n"
-                       "    Number of 1D Basis Nodes (p)       : %" CeedInt_FMT "\n"
-                       "    Number of 1D Quadrature Points (q) : %" CeedInt_FMT "\n"
-                       "    Global nodes                       : %" PetscInt_FMT "\n"
-                       "    DoF per node                       : %" PetscInt_FMT "\n"
-                       "    Global DoFs                        : %" PetscInt_FMT "\n",
-                       used_resource, CeedMemTypes[mem_type_backend], P, Q,
+                       "    Number of 1D Basis Nodes (p)            : %" CeedInt_FMT "\n"
+                       "    Number of 1D Quadrature Points (q)      : %" CeedInt_FMT "\n"
```

Comment:
Since this isn't strictly 1d, we probably should update the language here?

Reply:
Yeah, I think it was supposed to mean that we have Q=4 in 1D so that is not total quadrature point in 3D. I can erase ID.

Reply:
If I change it here, I need to change it in all other places in `example/petsc`, what is your suggestion?

Reply:
I think we have the Solution Order (P) and Quadrature Order (Q), right?

q_extra isn't really an extra number of quadrature points when you start talking about more general elements but rather additional order(s) of accuracy for the quadrature. We should tidy up this discussion in the Ratel docs too.

Reply:
Yeah, those names are better, I'll change them.